lib/mayaUsd/ufe/UsdTransform3dHandler.cpp

```diff
@@ -44,6 +46,14 @@ Ufe::Transform3d::Ptr UsdTransform3dHandler::transform3d(const Ufe::SceneItem::P
     assert(usdItem);
 #endif
 
+    // According to USD docs, editing scene description via instance proxies and their properties is
+    // not allowed.
+    // https://graphics.pixar.com/usd/docs/api/_usd__page__scenegraph_instancing.html#Usd_ScenegraphInstancing_InstanceProxies
+    if (usdItem->prim().IsInstanceProxy()) {
+        MGlobal::displayError("Authoring to an instance proxy is not allowed.");
+        return nullptr;
+    }
+
```

Comment:
This code is duplicated in 2 places. Would be nice to refactor it. Perhaps as part of "lib/mayaUsd/ufe/Utils"?

Reply:
Agreed.  Perhaps when we collect more illegal cases, we can do the refactoring.